src/app/modules/spcp/spcp.service.ts

```diff
@@ -178,8 +178,8 @@ export class SpcpService {
     } else {
       // The error page should have text like 'System Code:&nbsp<b>138</b>'
       const errorCode = getSubstringBetween(
-        loginHtml,
-        'System Code:&nbsp<b>',
+        loginHtml.toLowerCase(),
+        'System code:&nbsp;<b>'.toLowerCase(),
```

Comment:
nit: can't we just hardcode the lowercased version of this string haha
```suggestion
        'system code:&nbsp;<b>',
```